sktime/forecasting/base/tests/test_fh.py

```diff
@@ -319,7 +319,7 @@ def test_get_duration(n_timepoints, index_type):
             _make_index(n_timepoints, index_type)
 
 
-FIXED_FREQUENCY_STRINGS = ["10T", "H", "D", "2D"]
+FIXED_FREQUENCY_STRINGS = ["10H", "H", "D", "2D"]
```

Comment:
```suggestion
FIXED_FREQUENCY_STRINGS = ["10min", "H", "D", "2D"]
```

From `pandas` docs

![image](https://github.com/sktime/sktime/assets/39331844/107e400f-ad3d-4b86-97a7-2097d5e1c6c0)